osu.Game/Screens/OnlinePlay/OnlinePlayScreen.cs

```diff
@@ -99,6 +99,7 @@ private void forcefullyExit()
             Logger.Log($"{this} forcefully exiting due to loss of API connection");
 
             // This is temporary since we don't currently have a way to force screens to be exited
+            // See also: `DailyChallenge.forcefullyExit()`
```

Comment:
Readers will note that this copy of this logic contains an `else` branch that claims that it is "[handling] the case where a child screen is current (ie. gameplay)."

Dear reader, it does not, because the invocation of this method is copiously scheduled:

https://github.com/ppy/osu/blob/aded31bd071dc49c1a1be1af015915310a930f36/osu.Game/Screens/OnlinePlay/OnlinePlayScreen.cs#L78-L82

which means it will _not get called_ if the screen is not current. So this just doesn't work. This particular oopsie blames back to 5b405abc5253480414eb421028d48c1cb019ef0b which means I am not touching it with a ten foot pole here. (Unless you'd like me to remove it?)